person.rb

```diff
@@ -14,11 +28,7 @@ def of_age
     @age >= 18
   end
 
-  public
-
-  def can_use_services
-    return unless @age >= 18 || parent_permission
-
-    true
+  def add_rentals(book, date)
+    @rentals << Rental.new(date, book, self)
   end
```

Comment:
- [ ] This association method is okay but we still need to remove the `@rentals <<` part as that has been handled by the Rental constructor instead

  ```
  def add_rental(date, book)
    Rental.new(date, book, self)
  end
  ```